src/Sylius/Bundle/AttributeBundle/Resources/views/attributeChoice.html.twig

```diff
@@ -10,9 +10,9 @@
                     <div class="list-group row">
```

Comment:
Good catch, I think we have one more coupling issue in the CompilerPass. It assumes there is ``product_attribute`` service and that's incorrect.